### PR TITLE
[3.2.0] Adding Docs for Setting up a user for the Admin Portal 

### PR DESCRIPTION
**Pull request by Sarangan0219**
### Purpose

Fixes https://github.com/wso2/docs-apim/issues/1624

### CODE REVIEW

**Review by chamilaadhi — APPROVED** [review]

**Review by Mariangela — APPROVED** [review]

File: en/docs/administer/managing-users-and-roles/managing-users-for-admin-portal.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+2.  Click **Main**, and then click **Add** under **Users and Roles**.
+
+    ![Add users and roles]({{base_path}}/assets/img/administer/add-users-and-roles.png)
+
+3.  Click **Add New Role**.
+
+    ![Add new role]({{base_path}}/assets/img/administer/add-new-role.png)
+
+4. Enter the name of the user role which we are going to assign admin portal access (e.g : `adminPortalManager`) and click **Next**
+
```
Comment (Mariangela) [inline en/docs/administer/managing-users-and-roles/managing-users-for-admin-portal.md:26]: Image not showing
Screenshot 2020-11-27 at 13 50 33

File: en/docs/administer/managing-users-and-roles/managing-users-for-admin-portal.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+3. Provide the  role name which is created in step-1. Then click **Next** to proceed.
+
+    ![Add Role Mapping]({{base_path}}/assets/img/administer/add-new-role-mapping.png)
+
+4. In **Select permissions** select **Custom Permissions** and  navigate to  **Permissions** --> **admin**. 
+Assign the scopes according to the role of the user.
+
+    - You need to assign `apim:api_workflow_view`, `apim:api_workflow_approve`, `apim:admin_settings`, `apim:tenantInfo` to any user to login and access a minimum version of admin portal. A user with these scopes will get access to manage workflow tasks.  
+
+    ![Minimum Version of Admin Portal]({{base_path}}/assets/img/administer/minimum-version-admin-portal.png)
```
Comment (Mariangela) [inline en/docs/administer/managing-users-and-roles/managing-users-for-admin-portal.md]: Image not showing
Screenshot 2020-11-27 at 13 50 43

File: en/docs/administer/managing-users-and-roles/managing-users-for-admin-portal.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+1. Sign in to the management console (`https://<APIM_Host>:<APIM_Port>/carbon`) as the admin (default credentials are `admin`/`admin`).
+
+2.  Click **Main**, and then click **Add** under **Users and Roles**.
+
+    ![Add users and roles]({{base_path}}/assets/img/administer/add-users-and-roles.png)
+
+3.  Click **Add New user**.
+
+    ![Add admin manager user]({{base_path}}/assets/img/administer/add-manager-user.png)
```
Comment (Mariangela) [inline en/docs/administer/managing-users-and-roles/managing-users-for-admin-portal.md]: Image not showing
Screenshot 2020-11-27 at 13 50 50

File: en/docs/administer/managing-users-and-roles/managing-users-for-admin-portal.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+4.  Provide the username and password and click **Next**.
+
+5.  Select `adminPortalManager` and assign it to the user. Click **Finish** to complete.
+
+    ![Assign admin portal user role]({{base_path}}/assets/img/administer/add-admin-manager-role-to-user.png)
```
Comment (Mariangela) [inline en/docs/administer/managing-users-and-roles/managing-users-for-admin-portal.md]: Image not showing

**Review by Mariangela — CHANGES_REQUESTED** [review]

**Comment by Mariangela** [comment]
@Sarangan0219 , looks like you have forgotten to attach some images along with this PR. Please check my review comments. Also note that I updated your PR after reviewing it. I have sent the changes to your branch.

**Review by Mariangela — APPROVED** [review]

**Comment by Mariangela** [comment]
@Sarangan0219 , is this fix applicable to 3.0.0 and 3.1.0? Please send this fix to the master branch as well.

**Pull request merged**